Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -119,6 +119,16 @@ public class CommonOptionBag {
   public static readonly Option<bool> IncludeRuntime = new("--include-runtime",
     "Include the Dafny runtime as source in the target language.");
 
+  public static readonly Option<string> ReportFileOption = new("--report-file",
```

Comment:
Could you move these options to AuditCommand.cs?

Reply:
Will do!